### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by laurensbl**
Updated readme to bring it inline with the updated wiki. Removed some info from the readme that is more elaborately incorporated in the wiki (less is more).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,95 +1,41 @@
...
-If you want to test the client then use a test server:<br>
-`--server-url https://acme-staging.api.letsencrypt.org/directory`
+Java Runtime Environment (JRE) 8 or higher
 
-If you use Letsencrypt's production server for testing you can reach limits which are set by Letsencrypt (or your ACME provider).
+## Installation
 
-## Options:
-See all available options in wiki: https://github.com/porunov/acme_client/wiki/Options
+No installation needed. The application is packed into a single executable JAR file, just download or compile it and you're ready to go.
```
Comment (porunov) [inline README.md]: Can you change it to:

No installation needed. The application is packed into a single executable JAR file, just [download](../releases/) or [compile](./How-to-compile-PJAC-from-source) it and you're ready to go.

Like in wiki you did

**Comment by laurensbl** [comment]
Good point. Done.

**Comment by laurensbl** [comment]
Fixed minor inconsistency in `Supported functions` section.

**Comment by laurensbl** [comment]
Made another improvement to the `Supported functions` section. That should be all for today :)

**Pull request merged**